src/views/email.njk

```diff
@@ -6,9 +6,10 @@
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
     <form class="form" action={{ props.currentUrl }} method="post">
-      <input type="hidden" name="pageType" value={{ props.pageType }}>
-      
+      <input type="hidden" name="pageType" value={{ props.pageType }}>      
       {% include "includes/csrf_token.njk" %}
+      
+      {% include "includes/proposed-name.njk" %}  
```

Comment:
Can you update the tests to check that it is really present on the page?

Reply:
test added